### DISCUSSION TITLE
Would it be possible to show all objects in the background map that are also queried by quests?

### DISCUSSION THREAD

**Discussion by mcliquid** [streetcomplete/StreetComplete#5138] [Feedback]
When solving map notes, I notice that many notes are created because something seems to be missing in the map, although no quest is displayed anymore, because everything has already been answered.
This is often the case with benches, trash cans or similar nodes.

Would it be possible to configure the background map (JawgMaps) to show all the objects that would be queried by a quest?
I think this would reduce the amount of map notes that are created simply because of the visual non-existence of some objects.

One example:
Note: https://www.openstreetmap.org/note/3778306
Node: https://www.openstreetmap.org/node/1312184885

**Comment by FloEdelmann** [comment]
Might be better solved by https://github.com/streetcomplete/StreetComplete/issues/4912

**Reply by mcliquid** [reply]
But does this ticket really solve the problem I mentioned?
I think it has to do with the general visibility of objects on the map. Obviously some users think that a particular object does not yet exist in the database because it is not shown on the background map.

My concern is not that people want to be able to create this object in an overlay, but that notes are mistakenly created for an object that already exists but is not made visible on the map.

**Comment by matkoniecz** [comment]
Quests already show nearby existing relevant objects. That was not true in past.

How old version are used by this notes?

**Reply by mcliquid** [reply]
Regarding the note in my [example](https://www.openstreetmap.org/note/3778306) above it's StreetComplete 53.2 and thus the most up-to-date version today.

When I look at the [surroundings](https://www.openstreetmap.org/node/1312184885) from my [example](https://www.openstreetmap.org/note/3778306), only one bench has been placed there far and wide.
In this specific example, the bench was updated in OpenStreetMap only three months ago and `backrest=yes` is already tagged. So neither the "backrest" quest nor the "Is it still there?" quest remains for the user. The object is therefore not queried via any quest. Since it does not appear on the background map in StreetComplete, the user assumes that the object does not yet exist and creates a map note.

For this, and all other cases I have observed, it does not help if other objects (such as benches) are displayed in the vicinity, nor that you can add new benches via an overlay. I am purely asking about the display of existing objects (such as benches) in the normal background map view when answering quests :)

**Reply by mnalis** [reply]
Short answer: **No, you'd have to go ask Jawg people to implement that, and I'm very confident they would turn you down**. One or two specific important POIs (like e.g. `emergency=assembly_point`) which are not densely used, maybe; something as unimportant as bench (much lets thousand of similar tags), I'd say not likely in the least.

The point of using Jawg maps as **_background_** is that it is **very selective in what it shows and thus unobtrusive and not overwhelming** for users. That is the whole purpose of background map, to remove all but most important stuff away, so those can be shown on-demand by other layers. Having **everything** shown on the background ("base") map would wreak havoc to that idea of map being usable to regular target user.

That being said, [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) fork of StreetComplete does include option to show many (if not all?) POIs (cut into several categories) on the map all the time, if you want so. It does however make map look overstuffed (especially in urban setting), but it is useful sometimes (but relatively rarely, which is why I keep them enabled only in one of several of my quick-presets). Feel free to try it, though.


Anyway, what you ask is AFAIK **not possible to do using Jawg maps, as they don't include all OSM elements**. One could of course build their own vector tile server including and rendering _"everything"_, but (apart from that being a lot of a job to create icons for every existing tag out there, and costs/bandwidth associated with hosting that) that then also begs the question why not instead create raw-downloaded-OSM-elements renderer (like OsmAnd does, although in mostly-offline fashion) and drop a need for vector maps completely. 

(and this is what mentioned street furniture overlay https://github.com/streetcomplete/StreetComplete/issues/4912 would partly do -- so to accomplish what you want, you would simply enter that overlay and **never leave it**, and thus the map would display most such POIs all the time; which is I believe it was mentioned to you, as that is withing technical possibility of SC to display)

**Reply by mcliquid** [reply]
Thank you for the answer. In principle, this is exactly what answers my question, so it is not possible to customize the map 👍

**Reply by Helium314** [reply]
Though SC may switch to [directly rendering](https://github.com/streetcomplete/StreetComplete/issues/3123#issuecomment-1307972352) OSM data at some point, at least on high zoom levels.
This would help here, and with all the _outdated map_ issues
